cgi-bin/DW/External/Site.pm

```diff
@@ -218,6 +225,8 @@ sub get_site {
     return $mapped || undef;
 }
 
+sub get_deadsites { return %deadsites; }
```

Comment:
If this is going to be a function, I would just declare the hash inside the function since we're not using it in any other context:

```
sub get_deadsites {
  return (
              "del.icio.us" => 1,
              "diigo.com"   => 1,
              "imzy.com"    => 1,
              "inksome.com" => 1,
              "journalfen.net" => 1
  );
}
```